pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -424,13 +424,13 @@ func (m *ManagerImpl) Register(ctx context.Context, r *pluginapi.RegisterRequest
 	}
 	if !versionCompatible {
 		errorString := fmt.Sprintf(errUnsupportedVersion, r.Version, pluginapi.SupportedVersions)
-		klog.Infof("Bad registration request from device plugin with resource name %q: %s", r.ResourceName, errorString)
+		klog.InfoS("Bad registration request from device plugin with resource.", "resource", r.ResourceName, "error", errorString)
 		return &pluginapi.Empty{}, fmt.Errorf(errorString)
 	}
 
 	if !v1helper.IsExtendedResourceName(v1.ResourceName(r.ResourceName)) {
 		errorString := fmt.Sprintf(errInvalidResourceName, r.ResourceName)
-		klog.Infof("Bad registration request from device plugin: %s", errorString)
+		klog.InfoS("Bad registration request from device plugin", "error", errorString)
```

Comment:
```suggestion
		klog.InfoS("Bad registration request from device plugin", "err", errorString)
```

Reply:
We should pass the real error (`error` interface) in to the `err` key instead of message. Please move construction of error `fmt.Errorf(errorString)` above and pass it to InfoS